onnxruntime/core/providers/cuda/tensor/split.cc

```diff
@@ -76,6 +76,25 @@ Status SplitKernel::ComputeInternal(OpKernelContext* ctx) const {
   auto input_dims = input_shape.GetDims();
   auto output_dimensions{input_shape.AsShapeVector()};
 
+  if (split_sizes.size() == 3 && (axis == 2)) {
```

Comment:
Could you add a comment here?
This should work mostly for split QKV in transformer models.